crates/agglayer/tests/snapshots/config__config_display.snap

```diff
@@ -45,8 +45,9 @@ private-keys = []
 [telemetry]
 prometheus-addr = "0.0.0.0:3000"
 
-[epoch.time-clock]
-epoch-duration = 60
+[epoch.block-clock]
+epoch-duration = 8
```

Comment:
May be my Rust biases speaking but duration invokes real time rather than blocks. How about something like `blocks-per-epoch`? Hmm, maybe that's not better... idk.

Reply:
Yeah I too had some back and forth between both options. I end up letting it with `duration` as we refer to the `duration` of an epoch when discussing. And the `blocks` are here to have a global clock when we'll have multiple nodes. The goal was to define that if one block is 15sec, then if we want to have epoch that are 1h duration, we need 250 block duration.

Reply:
Ok. No strong opinion here, we can leave it as is.